ledger/participant-integration-api/src/main/resources/db/migration/postgres/V130__etq_data_migration.sql

```diff
@@ -0,0 +1,105 @@
+------------------------------------ ETQ Data migration -------------------------------
+
+-- Removes all elements from a that are present in b, essentially computes a - b.
+CREATE OR REPLACE FUNCTION etq_array_diff(a int[], b int[])
+RETURNS int[]
+AS
+$$
+      SELECT coalesce(array_agg(el), '{}')
+      FROM unnest(a) as el
+      WHERE el <> all(b)
```

Comment:
Do you have some data-points as to how expensive this could be when all records have to be rewritten in this manner?

Reply:
No, I haven't been checking that

Reply:
I think we need to do this to avoid nasty surprises in the future:
https://digitalasset.atlassian.net/wiki/spaces/DAMLX/pages/2415395897/DB+schema+migration+speed